website/src/pages/index.js

```diff
@@ -56,7 +56,7 @@ function Hero() {
                     </div>
                     <div className={styles.codeBlock}>
                         <CodeBlock className="language-bash">
-                            pipx run crawlee create my-crawler
+                            pipx run crawlee[cli] create my-crawler
```

Comment:
please add single quotes, because of zsh

```suggestion
                            pipx run 'crawlee[cli]' create my-crawler
```